CONTRIBUTING.md

```diff
@@ -1 +1,82 @@
-# Insert the contributing instructions here
+# Contributing instructions
+
+Thank you for considering to contribute to this repository!
+Here you will find instructions and guidelines for contributing efficiently.
+
+Before proceeding further, please review our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Issues
+
+If you find an issue in the [Schema.gov.it](https://www.schema.gov.it)
+website or in this repository, please check if it's already known by searching
+the [issue section](https://github.com/teamdigitale/dati-semantic-frontend/issues), otherwise
+file a new one: we really appreciate it! :rocket:.
+
+You're welcome to contribute to open issues with more information or by adding
+:+1: on them: it will help the maintainers identify the issues to be prioritized.
+
+### Creating a new issue
+
+There are three templates for new issues: `Bug report`, `Feature request`
+and `General issue`, pick the relevant one and follow the instructions
+in that template.
+
+If you pick `General issue` please provide a good amount of detail in
+order to let other people clearly understand the issue.
+After the creation of the issue, the maintainers team will promptly triage
+it by assigning the proper labels.
```

Comment:
We don't have template issues on this repo, we should rewrite or remove this.

Reply:
Removed.